packages-exp/auth-exp/src/core/errors.ts

```diff
@@ -340,18 +341,28 @@ export interface NamedErrorParams {
   serverResponse?: object;
 }
 
-type AuthErrorParams = {
```

Comment:
How will we support custom messages?

Reply:
no clue, let's check in with Fei about this